readme.md

```diff
@@ -191,6 +240,18 @@ Execute a file synchronously.
 
 Returns or throws a [`childProcessResult`](#childProcessResult).
 
+### \$(options?)\`command\`
+
+Same as `execa()` except both file and arguments are specified in a single tagged template string. For example, ``$`echo unicorns` ``is the same as `execa('echo', ['unicorns'])`.
+
+The [`shell` option](#shell) must be used if the `command` uses shell-specific features (for example, `&&` or `||`), as opposed to being a simple `file` followed by its `arguments`.
```

Comment:
We might want to add that quotes, backslashes and spaces are automatically escaped and have no special meaning unless the `shell` option is used. I can see this becoming a common mistake. About half of the issues in this repository are related to confusion about shell syntax and escaping.